src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -1360,23 +1360,32 @@ def ngens(self):
         """
         return 1
 
-    def random_element(self, degree=(-1,2), *args, **kwds):
+    def random_element(self, degree=(-1, 2), monic=False, *args, **kwds):
         r"""
-        Return a random polynomial of given degree or with given degree bounds.
+        Return a random polynomial of given degree (bounds).
 
         INPUT:
 
-        -  ``degree`` - optional integer for fixing the degree
-           or a tuple of minimum and maximum degrees. By default set to
-           ``(-1,2)``.
+        -  ``degree`` - (default: ``(-1, 2)``) integer for fixing the degree or
+           a tuple of minimum and maximum degrees
+
+        -  ``monic`` - optional boolean to indicate whether the sampled
+           polynomial should be monic, or not. If this is set, `0` is not a
+           possible output of the method
 
         -  ``*args, **kwds`` - Passed on to the ``random_element`` method for
            the base ring
```

Comment:
```suggestion
        -  ``degree`` — (default: ``(-1, 2)``) integer for fixing the degree or
           a tuple of minimum and maximum degrees

        -  ``monic`` — boolean (default: ``False``) whether the sampled
           polynomial should be monic

        -  additional (keyword) arguments are passed on to the ``random_element``
           method for the base ring
```
Since `0` is not a monic polynomial, it doesn’t need to be stated that it won’t return `0`.